README.md

```diff
@@ -30,11 +30,26 @@ not something else.
 
 Like `tmux`, it requires `libevent2` and `libtinfo` (usually packaged with ncurses).
 
+For Linux
+
 ```sh
 sudo apt-get install libncurses-dev libevent-dev
 cargo install tmux-rs
 tmux-rs
 ```
 
+For OSX
+```sh
+brew install libevent ncurses pkg-config
+
+# Set  platform specific path overrides 
+export PKG_CONFIG_PATH="$(brew --prefix)/lib/pkgconfig"
+export LIBRARY_PATH="$(brew --prefix)/lib"
+export CPATH="$(brew --prefix)/include"
```

Comment:
Only `LIBRARY_PATH` is needed. tmux-rs doesn't use C includes, so `CPATH` is not needed. pkg-config is already configured to use the directory where `libevent_core.pc` is located.

Even `LIBRARY_PATH` won't be needed once #45 is merged.